### DISCUSSION TITLE
How to manage the lifetime of data in the cache?

### DISCUSSION THREAD

**Discussion by Anton-Zubarev** [ImmediatePlatform/Immediate.Cache#7] [General]
How to manage the lifetime of data in the cache?

**Comment by viceroypenguin** [comment]
Ah, thanks for something I should add to the documentation.

In your `GetValueCache` class, override the `GetCacheEntryOptions()` method to return a `MemoryCacheEntryOptions` that has the lifetime you prefer (sliding expiration, absolute expiration, etc.)
